src/pages/login/CoreConnect.tsx

```diff
@@ -65,8 +69,51 @@ const CoreConnect = () => {
       });
   };
 
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch('http://132.145.101.179:42069/');
```

Comment:
We want a more robust way of checking this.
Like, if my server goes down, people will get random popups

Reply:
any suggestions